src/Games/NexusMods.Games.FOMOD/CoreDelegates/UiDelegate.cs

```diff
@@ -280,7 +291,9 @@ public void UpdateState(FomodInstaller.Interface.ui.InstallerStep[] installSteps
         if (CurrentArchiveFiles is null) return null;
 
         var asPath = RelativePath.FromUnsanitizedInput(image);
-        var node = CurrentArchiveFiles.FindByPathFromRoot(asPath);
+        var node = CurrentArchiveFiles.Path() != RelativePath.Empty
+            ? CurrentArchiveFiles.FindByPathFromRoot(asPath)
+            : CurrentArchiveFiles.FindByPathFromChild(asPath);
```

Comment:
You should add a short comment explaining this.

Reply:
I think I will change it to always  be `FindByPathFromChild`. The path of the image should always be relative to the root anyways, so root should never be considered, even if present.